simmechanics/data/hand_mk3/convertModel.sh

```diff
@@ -0,0 +1,2 @@
+simmechanics_to_urdf SIM_LEFT_HAND_Mk3.xml --yaml simmechanics2urdf_configfile.yaml --csv-joints simmechanics2urdf_joints_configfile.csv --output xml --outputfile left_hand_mk3.urdf
```

Comment:
Is this use? The bot will use the CMake infrastructure to generate the models, so to avoid confusion it could make sense to remove this (see https://github.com/robotology/icub-models-generator/pull/201).